src/components/user/uc/users.uc.integration.test.js

```diff
@@ -1,5 +1,6 @@
 const sinon = require('sinon');
 const { expect } = require('chai');
+const { ObjectId } = require('mongoose').Types;
```

Comment:
We can also use `generateObjectId` from the testObjects. Currently it is the same implementation, but by encapsulating the mongoose dependecy it will be easier to exchange it later if needed.